account_invoice_merge/models/account_move.py

```diff
@@ -107,7 +108,11 @@ def make_key(br, fields):
                     or field == "tax_ids"
```

Comment:
I don't like this `make_key` implementation.
Instead of checking value types with getattr, I would use smth like `type(br._fields[field]).
There, depending on the type (Selection, Many2many,…) you can decide what to do.
```python
if type(br._fields[field]) == fields.Many2many:
    records = br.get(field)
    value = set(records.ids) if records else False
```
You can do that for the special cases, but for most of them, you only need to use:
```python
value = br.get(field, False)
```

Reply:
Also, I would say that this `make_key` func should be extracted, and tested.

Reply:
@mmequignon i have refactored that function